### ISSUE TITLE
Normalization2D

### ISSUE THREAD

**Issue by hashansl** [keunwoochoi/kapre#100]
from kapre.utils import Normalization2D 
I'm getting a module not found error. Could you please help

**Comment by keunwoochoi** [comment]
it was removed from 0.3.0 (https://kapre.readthedocs.io/en/kapre-0.3.3/release_note.html). I believe 0.2.0 is the latest that has that layer.

**Comment by seth814** [comment]
I used tensorflow.keras.layers.LayerNormalization as a replacement.
